docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+wait-for-it "${DB_HOST}:${DB_PORT}"
```

Comment:
It's an ok solution, but you can use a `healthcheck` in docker-compose.
See: https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck